examples/tutorial/jupyter/execution/omnisci_on_native/README.md

```diff
@@ -0,0 +1,28 @@
+# Jupyter notebook examples to run Modin on OmniSci
+
+## Creating a development environment
+
+To get required dependencies for these Jupyter Notebooks
+you should create a development environment with `conda`
+using `jupyter_omnisci_env.yml` file located in the current directory:
+
+```bash
+conda config --set channel_priority strict
+conda env create -f jupyter_omnisci_env.yml
```

Comment:
Does OmniSci available on Windows? If not, we should explicitly highlight it at the beginning of the readme

Reply:
It doesn't, we have some info related to OmniSci on the main readme and probably somewhere else. I think it is enough.

Reply:
I don't think it's enough. For example, I wasn't sure whether I can run this notebook on Windows until I checked the supported OSs by modin-omnisci package. For a user who just reached the notebooks from some external place it could be confusing that he can't use them on Windows, unless there's an explicit note about it

Reply:
Ok, if there is a concern/misunderstanding, let's add a note.

Reply:
@YarShev @dchigarev I added `Notes` about this.